e2e/utils.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	// _ "github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1alpha1"                             // nolint
 	// _ "github.com/kubernetes-csi/external-snapshotter/pkg/client/clientset/versioned/typed/volumesnapshot/v1alpha1" // nolint
+	"github.com/ceph/ceph-csi/internal/util"
```

Comment:
Add an empty line between 'sections' of imports, that should address the CI complaint.

Reply:
> Add an empty line between 'sections' of imports, that should address the CI complaint.

Yes, This was the issue... Thanks!